firmware/config/boards/kinetis/config.mk

```diff
@@ -1,9 +1,12 @@
-
 #default=1536
 USE_PROCESS_STACKSIZE = 850
 #default=4096
 USE_EXCEPTIONS_STACKSIZE = 2400
 
+USE_FATFS = no
+USE_BOOTLOADER = no
+DEBUG_LEVEL_OPT = -O2
```

Comment:
we don't need DEBUG_LEVEL_OPT here - -O2 is the default

Reply:
Without this def the linker fails:
```
Linking build/rusefi.elf
lto-wrapper: warning: Options to '-Xassembler' do not match: -alms=build/lst/check.lst, -alms=build/lst/crt1.lst, dropping all '-Xassembler' and '-Wa' options.
/tmp/cc6zLWwe.s: Assembler messages:
/tmp/cc6zLWwe.s:18: Warning: ignoring changed section attributes for .data
/tmp/cc6zLWwe.s:34: Warning: dwarf line number information for .data ignored
...

/tmp/cc6zLWwe.s:10433: Warning: dwarf line number information for .data ignored
/tmp/cc6zLWwe.s:10437: Warning: dwarf line number information for .data ignored
/tmp/cc6zLWwe.s:10440: Warning: dwarf line number information for .data ignored
/tmp/cc6zLWwe.s:97651: Error: leb128 operand is an undefined symbol: .LVU25
/tmp/cc6zLWwe.s:97652: Error: leb128 operand is an undefined symbol: .LVU25
/tmp/cc6zLWwe.s:97670: Error: leb128 operand is an undefined symbol: .LVU15
/tmp/cc6zLWwe.s:97671: Error: leb128 operand is an undefined symbol: .LVU17
/tmp/cc6zLWwe.s:97672: Error: leb128 operand is an undefined symbol: .LVU17
/tmp/cc6zLWwe.s:97673: Error: leb128 operand is an undefined symbol: .LVU36
/tmp/cc6zLWwe.s:97674: Error: leb128 operand is an undefined symbol: .LVU36
/tmp/cc6zLWwe.s:97675: Error: leb128 operand is an undefined symbol: .LVU43
/tmp/cc6zLWwe.s:97676: Error: leb128 operand is an undefined symbol: .LVU43
/tmp/cc6zLWwe.s:97705: Error: leb128 operand is an undefined symbol: .LVU5
/tmp/cc6zLWwe.s:97706: Error: leb128 operand is an undefined symbol: .LVU11
/tmp/cc6zLWwe.s:97707: Error: leb128 operand is an undefined symbol: .LVU11
/tmp/cc6zLWwe.s:97726: Error: leb128 operand is an undefined symbol: .LVU2480
/tmp/cc6zLWwe.s:97727: Error: leb128 operand is an undefined symbol: .LVU2480
/tmp/cc6zLWwe.s:97728: Error: leb128 operand is an undefined symbol: .LVU2482
/tmp/cc6zLWwe.s:97729: Error: leb128 operand is an undefined symbol: .LVU2482
/tmp/cc6zLWwe.s:97730: Error: leb128 operand is an undefined symbol: .LVU2483
/tmp/cc6zLWwe.s:97731: Error: leb128 operand is an undefined symbol: .LVU2483
/tmp/cc6zLWwe.s:97764: Error: leb128 operand is an undefined symbol: .LVU2464
/tmp/cc6zLWwe.s:97765: Error: leb128 operand is an undefined symbol: .LVU2473
/tmp/cc6zLWwe.s:97766: Error: leb128 operand is an undefined symbol: .LVU2475
/tmp/cc6zLWwe.s:97783: Error: leb128 operand is an undefined symbol: .LVU2465
/tmp/cc6zLWwe.s:97784: Error: leb128 operand is an undefined symbol: .LVU2468
/tmp/cc6zLWwe.s:97793: Error: leb128 operand is an undefined symbol: .LVU2460
/tmp/cc6zLWwe.s:97794: Error: leb128 operand is an undefined symbol: .LVU2480
/tmp/cc6zLWwe.s:97795: Error: leb128 operand is an undefined symbol: .LVU2480
/tmp/cc6zLWwe.s:97796: Error: leb128 operand is an undefined symbol: .LVU2482
/tmp/cc6zLWwe.s:97797: Error: leb128 operand is an undefined symbol: .LVU2482
/tmp/cc6zLWwe.s:97821: Error: leb128 operand is an undefined symbol: .LVU51
lto-wrapper: fatal error: arm-none-eabi-gcc returned 1 exit status
compilation terminated.
/usr/lib/gcc/arm-none-eabi/11.3.1/../../../arm-none-eabi/bin/ld: error: lto-wrapper failed
collect2: error: ld returned 1 exit status
make: *** [ChibiOS/os/common/startup/ARMCMx/compilers/GCC/mk/rules.mk:217: build/rusefi.elf] Error 1
```

Reply:
huh, weird

Reply:
Yeah kind of strange ... including any `-g` option causes the warning (ignoring) and error.  Maybe some code/section mark can be fixed or something, unsure.